strongbox-commons/src/test/java/org/carlspring/strongbox/booters/SystemPropertiesInitializerTest.java

```diff
@@ -0,0 +1,174 @@
+package org.carlspring.strongbox.booters;
+
+import java.util.Properties;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+import org.springframework.context.annotation.Configuration;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.carlspring.strongbox.booters.SystemPropertiesInitializer.STRONGBOX_HOME_PROPERTY_NAME;
+import static org.carlspring.strongbox.booters.SystemPropertiesInitializer.STRONGBOX_STORAGE_BASEDIR_PROPERTY_NAME;
+import static org.carlspring.strongbox.booters.SystemPropertiesInitializer.STRONGBOX_VAULT_PROPERTY_NAME;
+
+class SystemPropertiesInitializerTest
```

Comment:
The thought crossed my mind "Is this test too excessive"? I don't think so. These "bootstrap" type pieces are key to a sane app and I think its valuable to test its functionality - happy paths and variants.